lib/rubocop/cop/style/stderr_puts.rb

```diff
@@ -17,12 +17,13 @@ module Style
       class StderrPuts < Cop
         include RangeHelp
 
-        MSG = 'Use `warn` instead of `$stderr.puts` to allow such output ' \
-              'to be disabled.'.freeze
+        MSG = 'Use `warn` instead of either `$stderr.puts` or `STDERR.puts` ' \
```

Comment:
I think in the message we should show exactly what we detected (meaning we should make the message dynamic).